SwiftJym/JymHomePage.swift

```diff
@@ -13,8 +13,8 @@ struct JymHomePage: View {
     //Workout categories
     private let muscleGroups = ["All", "Chest", "Back", "Shoulders", "Biceps", "Triceps"]
     
-    private let trendingExercises = ["deadlift1", "benchpress1", "squat1"]
-    private let recentExercises = ["crunches1", "seatedrow1", "dumbbellcurl1", "hammercurl1", "tricepdip1"]
+    private let trendingExercises = Exercise.trendingExample() // FIXME: injected an example array for preview/debug purpose
+    private let recentExercises = Exercise.recentExample() // FIXME: same as trendingExercises
```

Comment:
I needed to have an array of items instead of string (or a way to "translate" the exercise image name into the exercise name but I don't think that this last could be a good approach bcs in this scenario we still miss info like muscle)

Reply:
Yeah I see what you mean here, works for the test purposes so will still push the change but will look into the FIXMEs you've setup. Thanks